Unaccountable-Magpie/WorkforceManagement
# Jr setup

Purpose:
Shows a list of employees
How it fits in context of the project:
it was a ticket
Specific Feature affected:
Index view
How to test (Be thorough!):

1.Checkout into my branch jr-setup

2.Open BangazonApi.sln

3.click the green arrow button Execute

4. click on Employees in the nav bar and see...employees..?
